packages/lib-classifier/src/components/Classifier/components/withKeyZoom/withKeyZoom.spec.js

```diff
@@ -13,12 +13,18 @@ describe('withKeyZoom', function () {
   }
   const zoomStub = React.createRef()
   const WithZoom = withKeyZoom(StubComponent)
-  const onPan = sinon.stub()
+  const panLeft = sinon.stub()
+  const panRight = sinon.stub()
+  const panUp = sinon.stub()
+  const panDown = sinon.stub()
   const zoomIn = sinon.stub()
   const zoomOut = sinon.stub()
   const classifierStore = {
     subjectViewer: {
```

Comment:
Maybe create an instance of `subJectViewerStore` here, rather than mocking it with stubs?